recipe/meta.yaml

```diff
@@ -21,6 +21,8 @@ requirements:
     - {{ compiler('cxx') }}
     - bison
     - flex
+    - make
+  host:
```

Comment:
empty `host` stanza here is strange. Needed?

Reply:
IIRC it was needed to ensure that `build` and `host` were two different environments. This ensures installing the library doesn't clobber existing content from other packages used in the build, which would result in some content missing in the final package. That said, maybe conda-build has changed and this is no longer needed any more? Sorry it's hard to keep track.

Reply:
Yeah let's go ahead and back this out. Was waiting to see if the build would have other errors we needed to fix before changing, but [it looks ok]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=73706 ).